### PR TITLE
Requesting code review to merge to master

### PR DESCRIPTION
**Pull request by fedgut**

### CODE REVIEW

**Review by stratospherique — COMMENTED** [review]
### Pull request under review by Ahmed Mahfoudh TSE

File: index.html [hunk 1]
```diff
@@ -0,0 +1,918 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
```
Comment (stratospherique) [inline index.html:7]: - Replace the default viewport meta tag with the responsive one below :arrow_double_down: 
`<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">`

**Review by stratospherique — APPROVED** [review]
### Great work @fedgut :clap: . Your project checks all the requirements :100: . Congratulation, your project is approved :heavy_check_mark: and you can merge to master :+1: :+1: :+1: 

###### PS: please fix the issue below :arrow_down:

File: index.html (hunk 1 above)
Comment (fedgut) [reply index.html:7]: Thanks! Added the responsive meta

**Pull request merged**